src/Core/src/Core/IMenuElement.cs

```diff
@@ -7,12 +8,22 @@ public interface IMenuElement : IElement, IImageSourcePart, IText
 		/// <summary>
 		/// Represents a shortcut key for a MenuItem.
 		/// </summary>
+		[Obsolete]
```

Comment:
Can we remove all changes that happen in this PR? https://github.com/dotnet/maui/pull/14740

Reply:
Revert https://github.com/dotnet/maui/pull/14740?

Reply:
I went through and manually removed and/or swapped out all the changes. This seemed easier than to full on revert and reimplement